q2_demux/_summarize/_visualizer.py

```diff
@@ -204,6 +201,10 @@ def summarize(output_dir: str, data: _PlotQualView, n: int=10000) -> None:
 
     with open(os.path.join(output_dir, 'data.jsonp'), 'w') as fh:
         fh.write("app.init(")
+        fh.write('{"n":%s,"seqCount":%s,"minSeqLen":'
```

Comment:
Let's use `json` from stdlib to do this:

```python
        json.dump({'n': int(n), 'totalSeqCount': int(sequence_count),
                   'minSeqLen': min_seq_len}, fh)
```

Note, you'll need to cast `n` and `sequence_count` to `int` because `json` doesn't know how to serialize `numpy.int64`. Also, don't forget to also write out a `,` (comma) after emitting the JSON. I also think it would be more clear to rename `seqCount` to `totalSeqCount`.